extract/metadata.js

```diff
@@ -64,7 +64,12 @@ export default function metadata(parsedRootXml, manifest) {
   let mediaDurationData = [];
   let mediaNarratorData = [];
 
-  metadataInfo.meta.forEach(item => {
+  let metaTags = false;
+  if(metadataInfo.meta) {
+    metaTags = Array.isArray(metadataInfo.meta) ? metadataInfo.meta : [metadataInfo.meta];
+  }
+
+  metaTags && metaTags.forEach(item => {
```

Comment:
if `metaTags` isn't used outside of ` if(metadataInfo.meta) {`, shouldn't the forEach be inside the `if` then?